frontend/packages/topology/src/data-transforms/ModelContext.ts

```diff
@@ -209,6 +209,4 @@ export class ExtensibleModel {
   };
 }
 
-const ModelContext = createContext<ExtensibleModel>(null);
-
-export default ModelContext;
+export const ModelContext = createContext<ExtensibleModel>(null);
```

Comment:
I would think this should still be a default export but it's odd it was put into a variable first.

```
default export createContext<ExtensibleModel>(null);
```

Reply:
There are multiple exports.

Reply:
😕  Sure, but in these 3 lines, it didn't need to create a variable to do a `default export` -- but I think you handled this in your last change, so it's moot now.